easybuild/easyblocks/generic/configuremake.py

```diff
@@ -303,11 +305,19 @@ def configure_step(self, cmd_prefix=''):
             if host_type:
                 build_and_host_options.append(' --host=' + host_type)
 
+        if self.cfg.get('configure_without_installdir'):
+            configure_prefix = ''
+            if self.cfg.get('prefix_opt'):
+                print_warning("Specified prefix_opt '%s' is ignored due to use of configure_without_installdir",
+                              prefix_opt)
```

Comment:
I wonder if this should rather be an error than just a warning.... opinions??

Reply:
I'm happy with it as a warning.